### PR TITLE
feat: added grid component

### PR DESCRIPTION
**Pull request by TejasShekar**
## Grid Layouts added :
- 30-70 layout
- 50-50 layout
- 3 part layout

### Things to be noted : 
- Site not responsive yet.
- Other component link are broken as I have refactored them. Works fine locally. Will push it after this PR is reviewed.

### CODE REVIEW

**Review by ravikumar0403 — APPROVED** [review]
Code is clean and good to merge

File: Stylesheets/docs.css [hunk 1]
```diff
@@ -47,4 +47,13 @@
...
   color: var(--text-color);
+}
+
+.grid-demo{
+  display: flex;
+  align-items: center;
+  border: 1px solid var(--text-color);
+  border-radius: 2rem;
+  width: 550px;
+  height: 300px;
```
Comment (darshandamre) [inline Stylesheets/docs.css]: can use relative units here

**Review by darshandamre — APPROVED** [review]
i like how you have added multiple favicons for various sizes and devices

File: Stylesheets/docs.css (hunk 1 above)
Comment (TejasShekar) [reply Stylesheets/docs.css]: Yes. I have made the changes.

**Pull request merged**